docs/api.md

```diff
@@ -1368,6 +1368,7 @@ not created dynamically.
 | update_time | [google.protobuf.Timestamp](#google.protobuf.Timestamp) |  | Time of last tree update. Readonly (automatically assigned on updates). |
 | deleted | [bool](#bool) |  | If true, the tree has been deleted. Deleted trees may be undeleted during a certain time window, after which they&#39;re permanently deleted (and unrecoverable). Readonly. |
 | delete_time | [google.protobuf.Timestamp](#google.protobuf.Timestamp) |  | Time of tree deletion, if any. Readonly. |
+| index_bytes | [int32](#int32) |  | For a fixed-size data structure, the number of prefix bytes assigned to single-byte index shards. Not applicable to variable-size logs. Example, if the tree is a fixed-size data structure with 256 bits (32 bytes), and index_bytes=3: there will be 4 tiles to describe the path for any leaf; * 3 index tiles of 1 byte each; and * 1 final leaf tile of 29 bytes height. Readonly. |
```

Comment:
I find this name & description difficult to grok, "single-byte index shards".
I think it might be because it's jumping between the related topics of shards/indices/tiles.
Since this moves to a new map storage model of prefix strata and a single suffix mega-strata, perhaps could adopt those (or similar) terms and document (elsewhere) that a prefix stratum is an 8bit depth stratum and the suffix stratum is the remaining tree depth after the num_prefix_strata are taken into consideration?

Reply:
Addressed. PTAL.